src/ui/components/TransferGrain.js

```diff
@@ -18,102 +18,192 @@ export type Props = {|
         memo: memo,
       });
       setLedger(nextLedger);
       setAmount(fromInteger(0));
-      setSender(nextLedger.account(sourceIdentity.id));
+      setSender(nextLedger.account(sourceAccount.identity.id));
+      setReceiver(nextLedger.account(destAccount.identity.id));
       setMemo("");
     }
   };
 
   return (
-    <div
-      style={{
-        width: "80%",
-        margin: "0 auto",
-        background: "white",
-        padding: "0 5em 5em",
-      }}
-    >
-      <h1>Transfer Grain</h1>
-      <form onSubmit={(e) => submitTransfer(e)}>
-        <label>From</label>
-        <AccountDropdown ledger={ledger} setCurrentIdentity={setSender} />
-        <br />
-        <label>To</label>
-        <AccountDropdown ledger={ledger} setCurrentIdentity={setReceiver} />
-        <p>
-          <label htmlFor="amount">Amount</label> <br />
-          <input
-            type="number"
-            name="amount"
-            min="0"
-            step="any"
-            placeholder={`max: ${maxAmount}`}
-            required
-            value={amount}
-            onChange={(e) => setAmount(e.currentTarget.value)}
-          />
-          <span>{sourceIdentity && ` max: ${format(maxAmount, 5)}`}</span>
-        </p>
-        <p>
-          <label htmlFor="memo">Memo</label> <br />
-          <input
+    <form onSubmit={(e) => submitTransfer(e)}>
+      <div className={classes.root}>
+        <div className={classes.centerRow}>
+          <h1>Transfer Grain</h1>
+        </div>
+        <div className={classes.centerRow}>
+          <div
+            className={`${classes.dropdown} ${classes.centerRow} ${classes.third}`}
+          >
+            <AccountDropdown
+              ledger={ledger}
+              setCurrentIdentity={setSender}
+              placeholder="From..."
+            />
+          </div>
+          <div className={`${classes.centerRow} ${classes.third}`}>
+            <div className={classes.arrowBody}>
+              <div style={{width: "100px", display: "inline-block"}}>
+                <Input
+                  style={{color: "black", "-webkit-appearance": "none"}}
+                  disableUnderline
+                  type="number"
+                  name="amount"
+                  min="0"
+                  placeholder="(amount)"
+                  value={amount !== "0" ? amount : ""}
+                  onChange={(e) => setAmount(e.currentTarget.value)}
+                />
+              </div>
+              <span>
+                {sourceAccount &&
+                  sourceAccount.identity &&
+                  ` max: ${format(sourceAccount.balance, 2)}`}
+              </span>
+            </div>
+            <div className={classes.triangle} />
+          </div>
+          <div
+            className={`${classes.dropdown} ${classes.centerRow} ${classes.third}`}
+          >
+            <AccountDropdown
+              ledger={ledger}
+              setCurrentIdentity={setReceiver}
```

Comment:
finding it confusing that it looks like `setCurrentIdentity` actually takes an account, not an identity